### PR TITLE
Running PHP-CS-fixer to get Symfony coding standards

### PR DESCRIPTION
**Pull request by msvrtan**
This are small changes to get Symfony standard edition to obey symfony coding standard (defined by PHP-CS-fixer).

### CODE REVIEW

**Comment by hacfi** [comment]
@msvrtan Thanks for taking the time! Please create a separate PR for the changes of SymfonyRequirements.php at the repo https://github.com/sensiolabs/SensioDistributionBundle:
https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Resources/skeleton/app/SymfonyRequirements.php

File: app/autoload.php [hunk 1]
```diff
@@ -3,8 +3,8 @@
 use Doctrine\Common\Annotations\AnnotationRegistry;
 use Composer\Autoload\ClassLoader;
 
-/**
- * @var ClassLoader $loader
+/*
+ * @var ClassLoader
```
Comment (WouterJ) [inline app/autoload.php]: -1 for this change.

The old version was a completely valid doc block for the `$loader` variable. Seems like it isn't build into the PHP CS fixer yet.

The new version doesn't make any sense, as it's no longer a doc block and just a normal comment. No IDE will read it as being a docblock

File: app/check.php [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 $messages = array();
 foreach ($symfonyRequirements->getRequirements() as $req) {
-    /** @var $req Requirement */
+    /* @var $req Requirement */
```
Comment (WouterJ) [inline app/check.php]: same here

File: app/SymfonyRequirements.php [hunk 3]
```diff
@@ -119,10 +119,10 @@ class PhpIniRequirement extends Requirement
      *
      * @param string        $cfgName           The configuration name used for ini_get()
      * @param bool|callback $evaluation        Either a boolean indicating whether the configuration should evaluate to true or false,
-                                               or a callback function receiving the configuration value as parameter to determine the fulfillment of the requirement
```
Comment (Tobion) [inline app/SymfonyRequirements.php]: These changes must be done in SensioDistribution bundle as the header says.

File: app/autoload.php (hunk 1 above)
Comment (GrahamCampbell) [inline app/autoload.php]: If you oneline this phpdoc, then php-cs-fixer won't break it.

File: app/check.php (hunk 2 above)
Comment (GrahamCampbell) [inline app/check.php]: This regression has been fixed in php-cs-fixer now. It no longer makes this false change.

Comment (cordoval) [inline app/check.php]: i was actually liking this change :older_man:

Comment (GrahamCampbell) [inline app/check.php]: @cordoval this change is actually wrong. There is a difference between these two docblocks. The first one actually tokenises to something different to this one. This one is considered a comment, so all annotations have no meaning, where as, before it was considered a phpdoc comment, so annotations have meaning.

**Comment by Tobion** [comment]
Closing as most changes are wrong.

**Pull request closed without merging**